rust/operator-binary/src/connect/server.rs

```diff
@@ -398,8 +412,8 @@ pub(crate) fn build_service(
             .with_label(Label::try_from(("prometheus.io/scrape", "true")).context(LabelBuildSnafu)?)
             .build(),
         spec: Some(ServiceSpec {
-            type_: Some(service_type),
-            cluster_ip: service_cluster_ip,
+            type_: Some("ClusterIP".to_string()),
+            cluster_ip: Some("None".to_string()),
```

Comment:
I _think_ this causes extra allocations since ToString is a blanket impl.
_but I could be wrong_.

IMO, we should use `to_owned()` generally.

```suggestion
            type_: Some("ClusterIP".to_owned()),
            cluster_ip: Some("None".to_owned()),
```

Note: I'm unsure about the other `to_string` usages that come from Strum.

Reply:
In terms on consistency, we use `String::from` further below :shrug: